specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                                                                 |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                                                        |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Compute                   | Virtual Machines                  | Deployment and running of various applications in a flexible computing environment.                                                                                                                                                        |
+|                           | End User Computing                | Systems and tools that allow non-IT users to create and manage their own applications and data, including desktop virtualization, mobile computing, and self-service analytics.                                                            |
+|                           | Containers                        | Lightweight, portable units that package applications and their dependencies, ensuring they run consistently across different environments.                                                                                                |
+|                           | Serverless Compute                | Faciliates building and running applications without managing servers. Automatically scales resources based on demand, reducing operational complexity.                                                                                    |
+|                           | Quantum                           | Leverages the principles of quantum mechanics to perform complex calculations at high speed.                                                                                                                                               |
```

Comment:
"Quantum computing resources and simulators"

Reply:
How about:

> Resources and simulators that leverage the principles of quantum mechanics to perform certain calculations faster than classical computers.

Reply:
If we follow the way we did ServiceCategory descriptions, we didn't try to get into this level of specificity. This to me is a general feedback across all descriptions. Stay away from purpose. In this case, if that's what you want to go with, then 
> Resources and simulators that leverage the principles of quantum mechanics

Reply:
Sounds good; let's go with that.

```suggestion
|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                                               |
```